### PR TITLE
Project mobile version HTML and CSS file finished.

### PR DESCRIPTION
**Pull request by mm-pazouki**
Added portfolio HTML content.
Added CSS style for a mobile version.
Added Images and Icons of the project.

### CODE REVIEW

File: Contents/index.css [hunk 1]
```diff
@@ -0,0 +1,225 @@
+/* --------General------- */
+
+body {
+  background-color: white;
+  color: #1a2236;
+  margin: 0;
+  background-image: url(./Contents/Resources/image_geometry_2.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
```
Comment (alexpuente0) [inline Contents/index.css]: - [x] The decorative background images are not showing on the project. Please check this and make sure they do.
Your design VS Figma Template
image

File: Contents/index.css [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  letter-spacing: -0.01em;
+  text-align: justify;
+}
+
+h1 {
+  font-family: 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
```
Comment (alexpuente0) [inline Contents/index.css]: - [x] You are not using the fonts required in the template, please make sure you use the ones suggested (Poppins and Playfair Display in this case).

File: Contents/index.css [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+button.dark:hover {
+  background-color: white;
+  color: #1a2236;
+  border: 1px solid #1a2236;
+}
+
+button.dark:disabled {
+  background-color: gray;
+}
```
Comment (alexpuente0) [inline Contents/index.css]: - [x] Template requires a different style for a pressed button (bolder text and thicker border). Please make sure it works properly.

File: Index.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    </nav>
+  </header>
+
+  <section id="intro">
+    <h1>Hi, I'm <b>Mohammad Mehdi Pazouki</b>, Happy to see you</h1>
+    <p>
+      Hello! I am a software developer! I can help you build a product,
+      feature or website. Taka a look of my works. If you like what you see
+      and have a project you need coded, don't hesitate and contact me.
+    </p>
```
Comment (alexpuente0) [inline Index.html]: - [x] Temple requires you have only your name as a header, and then 2 paragraphs with the presentation text.  Please make sure you correctly follow the template as close as possible.

**Review by alexpuente0 — CHANGES_REQUESTED** [review]
Hi @mm-pazouki!



You've done a great job so far!

In this case, however, there are still some issues that you need to work on to finish this project and advance to the next one, but you are almost there!

## Highlights
- You have advanced a lot on your project, congratulations!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+      <h1>Mohammad Mehdi Pazouki</h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Take a look of my works. If you like what you see
+        and have a project you need coded, don't hesitate and contact me.
+      </p>
+      <button type="button" class="dark small">Start collaboration</button>
+      <div id="logo"></div>
+    </div>
+  </section>
```
Comment (Shisui6) [inline Index.html:65]: Nice job adding the background images as suggested by the previous reviewer👍. However, please make sure to stick to the design as much as possible. The decorative image at the bottom of the headline section should appear under the button as opposed to behind it. Please fix this🙏

Your design:
image

Figma design:
image

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @mm-pazouki, 

# Requires Changes 🍀
Great work on making the changes requested by a previous reviewer 👍
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet🪲.


 ## Highlights
 - Navbar and headline sections created ✅


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

- Please take a look at your readme file. It is missing important sections such as the `Getting Started` section and the `Future features` section. These sections are extremely important as they provide proper documentation for your project
- The link to your licence file is broken please take a look at it
- Please remember to always start your commit messages with a capital letter as it is considered best practice

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @mm-pazouki, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**